package.json

```diff
@@ -10,6 +10,7 @@
     "@dcl/content-hash-tree": "^1.1.3",
     "@dcl/crypto": "^3.4.5",
     "@dcl/hashing": "^3.0.4",
+    "@dcl/inspector": "^7.3.40",
```

Comment:
is it required to install the `@dcl/inspector` package or we could upgrade the `@dcl/sdk` package?

Reply:
ahh you are right, it’s the sdk we have to update